Makefile

```diff
@@ -5,9 +5,7 @@ DESTDIR ?= /usr/local/bin
 
 LIBC ?= musl
 
-# TODO: delete `KBC` parameter once KBC related functionalities are
-# all deprecated in code.
-KBC ?=
+PLATFORM ?=
```

Comment:
This is a bit confusing to me, especially given that we already have the `TEE_PLATFORM` parameter. Would it make more sense to call this `attester`?